src/velero/README.md

```diff
@@ -44,9 +44,47 @@ By overriding the velero values in the bundle as follows:
               value: "velero-bucket-credentials"
 ```
 
-## Plugin Compatibility
+#### Azure Blob Storage
 
-This package currently assumes the availability of S3 API compatible object storage. As such, only the AWS specific plugin image is included, in addition to the CSI plugin which is baked into Velero by default. More information about all available plugins [can be found in the upstream docs](https://velero.io/plugins/). Ironbank includes images for the Azure plugin, but it is currently excluded from this package. We may revisit package defaults at some point in the future depending on usage and user requests.
+Bucket information and access credentials can be provided via a secret with the following format.
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ###ZARF_VAR_VELERO_BUCKET_CREDENTIALS_SECRET###
+  namespace: velero
+type: kubernetes.io/opaque
+stringData:
+  cloud: |-
+    AZURE_STORAGE_ACCOUNT_ACCESS_KEY=${VELERO_STORAGE_ACCOUNT_ACCESS_KEY}
+    AZURE_CLOUD_NAME=${VELERO_CLOUD_NAME}
+```
+
+By overriding the velero values in the bundle as follows and adding the vlaues to the uds-config.yaml:
+```
+  - name: core
+    overrides:
+      velero:
+        velero:
+          values:
+            - path: credentials.secretContents.cloud
+              value: |       
+                AZURE_STORAGE_ACCOUNT_ACCESS_KEY=${VELERO_STORAGE_ACCOUNT_ACCESS_KEY}
+                AZURE_CLOUD_NAME=${VELERO_CLOUD_NAME}
+            - path: configuration.backupStorageLocation
+              value:
+                - name: default
+                  provider: azure
+                  bucket: ${VERLERO_BUCKET_NAME}
+                  config: 
+                    storageAccount=${VELERO_STORAGE_ACCOUNT}
+                    resourceGroup=${VELERO_RESOURCE_GROUP}
+                    storageAccountKeyEnvVar:${VELERO_STORAGE_ACCOUNT_ACCESS_KEY}
+```
```

Comment:
This section reads a bit confusing to me - I think the intent is that someone would just use bundle (helm) overrides to setup the azure connection, is that accurate? The way this reads it almost seems like I'd need to make my own secret first which I don't think is true? I think there's 2 separate options (existing secret someone manages on their own OR bundle overrides)?

Maybe not a part of this PR, but might be good down the road to re-evaluate where this doc belongs since it doesn't surface on the docs site if its just in this readme (cc @defenseunicorns/uds-core).

Reply:
@mjnagel You are correct, it's either or. I've updated it to just provide the bundle override method.